src/operator/optimizer_op-inl.h

```diff
@@ -372,8 +372,8 @@ inline void SGDMomUpdateDnsRspDnsImpl(const SGDMomParam& param,
         auto mom_data = mom.dptr<DType>();
```

Comment:
Got it. Will make it explicit next time. For this case, it should be obvious that it's a DType pointer.